src/v0/destinations/klaviyo/util.js

```diff
@@ -131,20 +90,7 @@ const checkForSubscribe = (message, traitsInfo, destination) => {
     (traitsInfo.properties?.listId || destination.Config?.listId) &&
```

Comment:
Here we are giving preference to event property(`traitsInfo`) over `destConfig.listId`. Is it ok ?

Reply:
Yes, this is fine